arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,214 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "SqlManagementClient",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete Databases, Servers and related resources.",
+        "version": "2015-05-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
+    ],
+    "produces": [
+        "application/json",
+        "text/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/securityAlertPolicies/Default": {
+            "put": {
+                "x-ms-long-running-operation": true,
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_CreateOrUpdateServerPolicy",
```

Comment:
The naming of the operationId should  be of the format {noun}_{verb} where noun is the operationGroup and verb is the action to be performed. This would translate to a call like this: 
``` client.SecurityAlertPolicy.CreateOrUpdate() v/s the current one: client.SecurityAlertPolicy.CreateOrUpdateServerPolicy(), where "ServerPolicy" seems redundant.
```